racingcar-java/src/test/java/domain/CarTest.java

```diff
@@ -0,0 +1,43 @@
+package domain;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import static org.assertj.core.api.Assertions.*;
+
+class CarTest {
+
+    Car c;
+
+    @BeforeEach
+    void init() {
+        c = new Car();
+    }
+
+    @Test
+    @DisplayName("move 성공 테스트")
+    void moveTest(){
+        int position = c.getPosition();
+        c.move(5);
+        assertThat(c.getPosition()).isEqualTo(position+1);
```

Comment:
```suggestion
        int nowPosition = c.move(5);
        assertThat(nowPosition).isEqualTo(position+1);
```
상황마다 다르겠지만 `c.move(5)`의 반환이 현재 poistion을 반환하게끔 한다면 굳이 getPosition을 호출하지 않아도 테스트할 수 있겠네요.